src/vis_cpu/vis_cpu.py

```diff
@@ -125,7 +133,7 @@ def vis_cpu(
                 # TODO: Try using a log-space beam for accuracy!
         else:
             # Primary beam pattern using direct interpolation of UVBeam object
-            az, za = conversions.lm_to_az_za(tx, ty)
+            az, za = conversions.lm_to_az_za(el=ty, m=tx) # args: (el=ty, m=tx)
```

Comment:
don't need the following comment I think?

Reply:
Removed, thanks.